superset-frontend/src/SqlLab/components/SqlEditorLeftBar/index.tsx

```diff
@@ -85,7 +85,9 @@ const collapseStyles = (theme: SupersetTheme) => css`
     padding: 0px ${theme.gridUnit * 4}px 0px 0px !important;
   }
   .ant-collapse-arrow {
-    top: ${theme.gridUnit * 2}px !important;
+    padding: ${theme.gridUnit / 2}px 0 !important;
+    bottom: ${theme.gridUnit}px !important;
+    right: ${theme.gridUnit * 4}px !important;
```

Comment:
Do we actually need those styles? It looks like the arrow is correctly centred when you just remove the padding (see screenshot).

<img width="1546" alt="image" src="https://github.com/apache/superset/assets/15073128/b27d1331-aea3-4447-b39f-6c7c35e099cd">

Reply:
Also, I wish we could clean up those `!important` statements... but we can leave that for another PR/task